pkg/key/app.go

```diff
@@ -141,17 +141,10 @@ func IsInOrgNamespace(customResource v1alpha1.App) bool {
 	return strings.HasPrefix(customResource.ObjectMeta.Namespace, "org-")
 }
 
-// IsManagedByFlux returns true if the giantswarm.io/managed-by label is set to
-// flux and is being validated by app-admission-controller. When true we skip
-// validating configmap and secret names. This simplifies managing these
-// resources with Flux. We still perform the validation in app-operator which
-// sets the App CR status.
-func IsManagedByFlux(customResource v1alpha1.App, projectName string) bool {
-	if projectName != "app-admission-controller" {
-		return false
-	}
-
-	return customResource.Labels[label.ManagedBy] == "flux"
+// IsManagedBy returns true if the giantswarm.io/managed-by label is set on the
+// resource. When true we skip validating configmap and secret names.
+func IsManagedBy(customResource v1alpha1.App) bool {
+	return customResource.Labels[label.ManagedBy] != ""
```

Comment:
We need to keep the check for app-admission-controller.

This is so in app-operator we wait until the resources actually exist.

Reply:
Ah sorry, my bad, I misread the intention, restoring it

Reply:
Can you restore the comment to? To explain why we need the damn thing 😄

Reply:
done!